libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -184,11 +184,13 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 	}
 	else
 	{
+		m_assembly.setSourceLocation(_varDecl.location);
```

Comment:
Is this needed, as there is another call after?

Reply:
It has to be called before the call to `appendConstant`